### PR TITLE
Adding view

### PR DESCRIPTION
**Pull request by auromota**
I added some HTML to the project. It is still very simple though. I also have changed how validation is made, because I think it should accept valid strings like "30" answell. Cya!

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = function(config) {
 
     // list of files / patterns to load in the browser
     files: [
-      'app/js/*.js',
+      'app/js/service/*.js',
```
Comment (edysegura) [inline karma.conf.js:18]: We can use this pattern instead:

```
'app/js/**/*.js'
```

File: test/FactorialServiceSpec.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ describe('FactorialService.js', function() {
         expect(FactorialService.calculate(2)).toBe(2);
         expect(FactorialService.calculate(3)).toBe(6);
         expect(FactorialService.calculate(4)).toBe(24);
-        expect(FactorialService.calculate(5)).toBe(120);
+        expect(FactorialService.calculate('5')).toBe(120);
```
Comment (edysegura) [inline test/FactorialServiceSpec.js:9]: Not valid in my option, calculate method should handle only number type.

Comment (edysegura) [inline test/FactorialServiceSpec.js:9]: Not valid in my option, calculate method should handle only number type.

**Comment by edysegura** [comment]
Hi @auromota!

The method calculate should only accept positive numbers. The controller is the one that should be responsable to parse string to number.

Before your commit the HTML was added by me! ;)

Thank you for colaborating!

**Comment by auromota** [comment]
Hi, @edysegura!

I saw you had committed the HTML immediately after I've done the pull request.

Thanks!

**Pull request closed without merging**